drivers/sx127x/sx127x_netdev.c

```diff
@@ -46,6 +46,7 @@ void _on_dio3_irq(void *arg);
 
 static int _send(netdev_t *netdev, const iolist_t *iolist)
 {
+    DEBUG("[sx127x] Sending packet now.\n");
```

Comment:
the commit reflect a "lorawan" change, but this is a driver change. I'm ok with this piggyback commit, but at least it should have its own scope

Reply:
Shall separate the commit :)